src/lib/components/Cytoscape.react.js

```diff
@@ -10,9 +10,9 @@ import _ from 'lodash';
 import CyResponsive from '../cyResponsive.js';
 
 /**
-A Component Library for Dash aimed at facilitating network visualization in
-Python, wrapped around [Cytoscape.js](http://js.cytoscape.org/).
- */
+  A Component Library for Dash aimed at facilitating network visualization in
+  Python, wrapped around [Cytoscape.js](http://js.cytoscape.org/).
+   */
```

Comment:
```suggestion
 * A Component Library for Dash aimed at facilitating network visualization in
 * Python, wrapped around [Cytoscape.js](http://js.cytoscape.org/).
 */
```
The extra spaces as you had them created weird whitespace in the generated back-end components. Leading `*` aligned with the first `*` in the first line is how we've formatted multi-line comments [elsewhere](https://github.com/plotly/dash/blob/d9b8d17e3fae5c7f9f342e480a85e782f6289fe8/components/dash-core-components/src/components/Checklist.react.js#L6-L11)